README.rst

```diff
@@ -27,7 +42,7 @@ To install from PyPI:
     pip3 install django-google-spanner
 
 
-To install from source:
+To install from the source:
```

Comment:
```suggestion
To install from source:
```

"Install from source" actually sounds more natural to me. I was curious, but don't know that this proves anything:

![image](https://user-images.githubusercontent.com/43799105/101409353-a1734980-3892-11eb-9ffa-91046588955a.png)

Reply:
👍 Done!